qutebrowser/misc/elf.py

```diff
@@ -316,21 +316,13 @@ def parse_webenginecore() -> Optional[Versions]:
     else:
         library_path = pathlib.Path(QLibraryInfo.location(QLibraryInfo.LibrariesPath))
 
-    filenames = [
-        # PyQt bundles those files with a .5 suffix, and Linux distributions
-        # most likely symlinks that.
-        'libQt5WebEngineCore.so.5',
-        # OpenBSD
-        'libQt5WebEngineCore.so.1.0',
-    ]
-    for filename in filenames:
-        lib_file = library_path / filename
-        if lib_file.exists():
-            log.misc.debug(f"QtWebEngine .so found at {lib_file}")
-            break
-    else:
+    library_name = sorted(pathlib.Path(library_path).glob('libQt5WebEngineCore.so*'))
```

Comment:
Haven't noticed that before, but `library_path` is already a `pathlib.Path` object, so the conversion here isn't needed:

```suggestion
    library_name = sorted(library_path.glob('libQt5WebEngineCore.so*'))
```